### PR TITLE
Parse/unint64s

### PR DESCRIPTION
**Pull request by sjeandeaux**
Add representation of UUID with two longs (uint64)

### CODE REVIEW

File: uuid.go [hunk 1]
```diff
@@ -122,6 +133,16 @@ func (uuid UUID) URN() string {
...
 }
 
+// MostSignificantBits the most significant 64 bits of this UUID's 128 bit value.
+func (uuid UUID) MostSignificantBits() uint64 {
+	return binary.BigEndian.Uint64(uuid[:8])
+}
+
+// LeastSignificantBits the least significant 64 bits of this UUID's 128 bit value.
+func (uuid UUID) LeastSignificantBits() uint64 {
+	return binary.BigEndian.Uint64(uuid[:8])
```
Comment (silas) [inline uuid.go]: Typo here, should be `uuid[8:]`

**Comment by pborman** [comment]
What is the use case for this addition?

**Comment by sjeandeaux** [comment]
In some world like java, we use the representation on two longs. I propose a parsing method allowing to pass from this representation to the UUID.

**Comment by sjeandeaux** [comment]
The documentation [java](http://docs.oracle.com/javase/7/docs/api/java/util/UUID.html#UUID(long,%20long))

**Comment by pborman** [comment]
Thanks for the reference @sjeandeaux.  This functionality can be completely done outside of the package.  I would think if you were interacting with Java that you probably already have a package that deals with Java <-> Go conversions.  This would be one of them.  I agree it would be convenient for you if they were in the Go package, but I am a bit troubled by adding in Java hooks.  We did add hooks for SQL, but those could not be implemented outside of the package.  Do you have any thought of how many other developers would be affected by this?

**Comment by sjeandeaux** [comment]
@pborman you're right. i close the pull-request.
I thank you for having taken the time to read this PR.

**Pull request closed without merging**